bind_exporter.go

```diff
@@ -534,8 +528,8 @@ func main() {
 			"Path to BIND's pid file to export process information",
 		).Default("/run/named/named.pid").String()
 		bindVersion = kingpin.Flag("bind.stats-version",
-			"BIND statistics version. Can be detected automatically.",
-		).Default("auto").Enum("json", "xml.v2", "xml.v3", "auto")
+			"BIND statistics channel",
+		).Default("json").Enum("json", "xml", "xml.v3", "auto")
```

Comment:
Seems like `auto` is also being removed.

Reply:
`auto` (and `json`) will be handled by the default switch case.
```go
	switch version {
	case "xml", "xml.v3":
		c = xml.NewClient(url, &http.Client{Timeout: timeout})
	default:
		c = json.NewClient(url, &http.Client{Timeout: timeout})
	}
```
The "auto" code was too tightly coupled to the XML client, and would have required a major rethink to support JSON. Since BIND 9.10 and later always support both XML v3 and JSON, the "auto" concept outlived its usefulness.